lib/auth/api.go

```diff
@@ -663,6 +663,43 @@ type WindowsDesktopAccessPoint interface {
 	accessPoint
 }
 
+// ReadDiscoveryAccessPoint is a read only API interface implemented by a certificate authority (CA) to be
```

Comment:
What does "implemented by a certificate authority (CA)" mean here?

Discovery has nothing to do with issuing certificates, right?